CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 -   **terra-stopwatch** has been removed.
 -   **terra-slider** has been removed.
+-   **interactjs** has been removed. Also all interfaces from `src/lib/components/interactables/` directory has been removed.
```

Comment:
Hier müsstest du schon etwas präziser sein.
Es wurden ja das `TerraInteractModule`, die `TerraDraggableDirective`, die Interfaces und die dependency zu  `interactjs` entfernt.
z.B.
```suggestion
-   **interactables** `TerraInteractModule`, `TerraDraggableDirective` and all related interfaces have been removed. See https://github.com/plentymarkets/terra-components/tree/v11.6.0/src/lib/components/interactables for reference.
-   **dependencies** removed peer dependency `interactjs` since its no longer used.
```